en/docs/install-and-setup/setting-up-databases/changing-default-databases/changing-to-ibm-db2.md

```diff
@@ -110,8 +110,8 @@ Follow the instructions below to change the type of the default datasource.
     [database.apim_db]
     type = "db2"
     url = "jdbc:db2://localhost:50000/reg_db"
```

Comment:
is this url correct?

Reply:
@chamilaadhi seems the db name is jumbled for configs in db2. I will fix this.

Reply:
@chamilaadhi this is fixed